src/archemist/core/state/optimisation_state.py

```diff
@@ -11,6 +11,7 @@ def from_dict(cls, state_dict: dict = {}):
         model = OptimisationModel()
         model.optimizer_module = state_dict['module']
         model.optimizer_class = state_dict['class']
+        model.optimizer_file_class_name = state_dict['Optimizer_file_class_name']
```

Comment:
you don't need this as we assume the optimiser class is loaded in the workspace. Please remove